LibertyDSNP/parquetjs
# Export the types into the package

Problem
=======

We want to be able to use types in external code.

Solution
========
Add the types to the build and export them in the package.

Completes most of #31 and #25 excepting releases.

Change summary:
---------------
* Export one additional class
* Remove source maps from the export
* Add type build to the publish step
* Update esbuild and two other packages for better ts builds and audit fixes

Steps to Verify:
----------------
1. `npm publish --dry-run && npm pack`
1. in an adjacent directory checkout the sdk-ts
1. in sdk: `npm i ../parquetjs/dsnp-parquetjs-0.0.0.tgz`
1. Run build and tests: `npm run build && npm run tes`
1. Should all pass!
